.github/workflows/ci.yml

```diff
@@ -20,11 +20,10 @@ jobs:
       - name: Download image
         run: |
           docker pull nfcore/rnaseq:dev && docker tag nfcore/rnaseq:dev nfcore/rnaseq:1.4
```

Comment:
```suggestion
          docker pull nfcore/rnaseq:dev && docker tag nfcore/rnaseq:dev nfcore/rnaseq:1.4.1
```

Reply:
This needs a fix for running tests 👍 

Also something that `nf-core bump-versions` should do soon 👍